### PR TITLE
docs: Add a new app and new github icon to showcase page

### PR DESCRIPTION
**Pull request by jvidalv**
Hi, I just want to share with you the open source app I've made using react-native-paper!

https://play.google.com/store/apps/details?id=josep.astrale

Cheers!

### CODE REVIEW

**Comment by jvidalv** [comment]
Seems like an update from last week is breaking the linter tests.

I'll create a separate commit to fix that.

**Comment by jvidalv** [comment]
I have added a new icon and a new property in order to be able to share the github repository in open source apps:

Preview

File: docs/pages/src/Showcase.js [hunk 1]
```diff
@@ -104,6 +112,15 @@ export default class Showcase extends React.Component<{}> {
                         {item.name}
                       </AppName>
                       <BadgeContainer>
+                        <a
```
Comment (Trancever) [inline docs/pages/src/Showcase.js:115]: @jvidalv Could you render githubIcon as a last icon instead of it being the first one?

Comment (jvidalv) [reply docs/pages/src/Showcase.js]: Done!

**Pull request merged**